test/functional/test-url-replacements.js

```diff
@@ -432,15 +432,15 @@ describes.sandboxed('UrlReplacements', {}, () => {
     it('should add extra params to SOURCE_URL', () => {
       const win = getFakeWindow();
       win.location = parseUrlDeprecated(
-          'https://cdn.ampproject.org/a/o.com/foo/?amp_r=hello%3Dworld');
+          'https://cdn.ampproject.org/a/o.com/foo/?a&amp_r=hello%3Dworld');
       sandbox.stub(trackPromise, 'getTrackImpressionPromise').callsFake(() => {
         return Promise.resolve();
       });
       return Services.urlReplacementsForDoc(win.ampdoc)
           .expandUrlAsync('?url=SOURCE_URL')
           .then(res => {
-            expect(res).to.contain(
-                encodeURIComponent('http://o.com/foo/?hello=world'));
+            expect(res).to.equal(
+                '?url=' + encodeURIComponent('http://o.com/foo/?hello=world&a='));
```

Comment:
@brandondiamond Is the trailing `=` right?

Reply:
Honestly I think we should stick the the format as best as we can.  But @lannka believes it makes sense to normalize the params if we need to change it anyway by add those extra params. Open to change the logic if there's any ideas.

Reply:
I don't believe there was a recommendation one way or another when we were chatting about implementing this. As long as we're consistent, we should be fine :+1: